### DISCUSSION TITLE
How to return string in double quotes, when using `Select Current Provider Path`

### DISCUSSION THREAD

**Discussion by shivanshu-semwal** [kelleyma49/PSFzf#171] [Q&A] [answered]
When using keycord `Ctrl+T`, if the file name have space then the string the returned surrounded by single quotes.
How can I change that to double quotes?

![image](https://user-images.githubusercontent.com/53912074/185050095-1e014481-d2e7-4647-b75c-c66304259d3a.png)

![image](https://user-images.githubusercontent.com/53912074/185050128-776529e5-123c-446f-96f6-d79cdd912f9d.png)

 I want double quotes because sometime files have single quote in their names.

**Comment by kelleyma49** [comment, marked as answer]
Fixed in https://www.powershellgallery.com/packages/PSFzf/2.5.6-alpha
